experimenter/experimenter/experiments/models.py

```diff
@@ -430,6 +438,7 @@ class NimbusExperiment(NimbusConstants, TargetingConstants, FilterMixin, models.
         blank=True,
         default=list,
     )
+    tags = models.ManyToManyField(Tag, blank=True, related_name="tags")
```

Comment:
Oh I don't think this is the right `related_name`.  That would be the field that appears on a tag instance to point back to experiments, I think you'd want something like `related_name="experiments"` here?